sysboundary/ionosphere.cpp

```diff
@@ -690,22 +690,22 @@ namespace SBC {
       // Now we have integrated density from the bottom of the atmosphere in the depth field.
       // Flip it around.
       for(int h=0; h<numAtmosphereLevels; h++) {
-         atmosphere[h].depth = integratedDensity - atmosphere[h].depth;
+         atmosphere.at(h).depth = integratedDensity - atmosphere.at(h).depth;
       }
 
       // Calculate Hall and Pedersen conductivity coefficient based on charge carrier density
       const Real Bval = 5e-5; // TODO: Hardcoded B strength here?
-      const Real NO_gyroFreq = physicalconstants::CHARGE * Bval / (31*physicalconstants::MASS_PROTON); // Ion (NO+) gyration frequency
```

Comment:
Here's one diff not due to my .at() craze.

Reply:
However, gumics also uses 31 here. I'm still not sure if that is not just simply wrong.